src/ui/common/src/utils/sidesheets.tsx

```diff
@@ -35,7 +35,7 @@ export function getDataSideSheetContent(
         px={'16px'}
         maxWidth="800px"
         height="100vh"
-        sx={{ overflowY: 'scroll' }}
+        sx={{ overflowY: 'hidden' }}
```

Comment:
Why is this changed? Are users still able to scroll vertically on the sidesheet if there is content to scroll to?

My hopes of doing things like this was to have the root layout handle showing scroll bars.

Reply:
@eunice-chan this may be related to the work that you are doing here: 
https://linear.app/aqueducthq/issue/ENG-1873/enable-scrolling-in-side-panel

Reply:
I misunderstood what 'hidden' means, updated to use 'auto'. I also added some minor adjustments so that scroll bar position and margin looks better:

<img width="1626" alt="Screen Shot 2022-10-19 at 5 24 17 PM" src="https://user-images.githubusercontent.com/10411887/196830104-18ba5b7d-d6aa-4e1d-881d-450b0f8500c4.png">